<?xml version="1.0"?>

<launch>

    
    <node name="vigir_l_arm_tuning"   pkg="sinusoidal_trajectory_widget"     type="sinusoidal_trajectory_widget"  output="screen" /> <!-- launch-prefix="xterm -e" / -->
    <param name="chain_file" value="$(find sinusoidal_trajectory_widget)/launch/"/>
    <param name="chain_name" value="l_arm"/>

</launch>

